lib/ansible/modules/cloud/podman/podman_image.py

```diff
@@ -0,0 +1,720 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+  module: podman_image
+  author:
+      - Sam Doran (@samdoran)
+  version_added: '2.8'
+  short_description: Pull images for use by podman
+  notes: []
+  description:
+      - Build, pull, or push images using Podman.
+  options:
+    name:
+      description:
+        - Name of the image to pull, push, or delete. It may contain a tag using the format C(image:tag).
+      required: True
+    executable:
+      description:
+        - Path to C(podman) executable if it is not in the C($PATH) on the machine running C(podman)
+      default: 'podman'
+      type: str
+    cert_dir:
```

Comment:
similar note as below for TLS standardization, at least aliases please https://github.com/ansible/ansible/issues/52506

Reply:
I went with `ca_cert_dir` even there is no mention of a directory for CA certs in that issue.